tests/test_wave_05.py

```diff
@@ -53,12 +53,17 @@ def test_new_genre_rec_from_empty_friends():
         ]
     }
 
-    raise Exception("Test needs to be completed.")
+    recommendations = get_new_rec_by_genre(sonyas_data)
+
+    assert not recommendations
+    assert len(sonyas_data["watched"]) == 1
```

Comment:
👍 Here, the function is specified as returning a list. Though in regular code, we would tend to write `if not my_list:` to check for an empty list, we can be more strict here to enforce the function requirements by literally comparing to `[]`.

```py
    assert recommendations == []
```

Further, we don't expect the `get_new_rec_by_genre` function to modify the data structure (in any case), so we don't need to check that the watched list is unchanged.